Makefile

```diff
@@ -77,54 +74,25 @@ test_postgres:
 	docker rm -f postgres-testrunner-postgres || true
 	docker run --name postgres-testrunner-postgres \
 		-e POSTGRES_PASSWORD=$(PG_PASSWORD) -e POSTGRES_USER=${PG_USER} \
-		-p 5432:5432 -d postgres:9.2
-	until pg_isready -d $(PG_HOST); do sleep 1; echo "Waiting for postgres"; done
+		-p ${PG_PORT}:5432 -d postgres:${PG_VERSION}
+	docker exec -it postgres-testrunner-postgres bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
+	docker exec -it postgres-testrunner-postgres bash -c "psql -U ${PG_USER} -d $(PG_PASSWORD) -c 'CREATE EXTENSION IF NOT EXISTS pg_trgm;'"
 	$(TEST_ENV) alembic upgrade head
-	$(TEST_ENV) py.test --timeout=7200 --verbose --show-count ./ --color=no \
-		--ignore=endpoints/appr/test/ -x
-	docker rm -f postgres-testrunner-postgres || true
-
-WEBPACK := node_modules/.bin/webpack
-$(WEBPACK): package.json
-	npm install webpack
-	npm install
-
-BUNDLE := static/js/build/bundle.js
-$(BUNDLE): $(WEBPACK) tsconfig.json webpack.config.js typings.json
-	$(WEBPACK)
-
-GRUNT := grunt/node_modules/.bin/grunt
-$(GRUNT): grunt/package.json
-	cd grunt && npm install
-
-JS  := quay-frontend.js quay-frontend.min.js template-cache.js
-CSS := quay-frontend.css
-DIST := $(addprefix static/dist/, $(JS) $(CSS) cachebusters.json)
-$(DIST): $(GRUNT)
-	cd grunt && ../$(GRUNT)
-
-build: $(WEBPACK) $(GRUNT)
-
-docker-build: build
-	ifneq (0,$(shell git status --porcelain | awk 'BEGIN {print $N}'))
-	echo 'dirty build not supported - run `FORCE=true make clean` to remove'
-	exit 1
-	endif
-	# get named head (ex: branch, tag, etc..)
-	NAME = $(shell git rev-parse --abbrev-ref HEAD)
-	# checkout commit so .git/HEAD points to full sha (used in Dockerfile)
-	git checkout $(SHA)
+	@echo $$(for i in {1..120}; do printf "-"; done)
+	@echo "postgres is ready to accept connections!"
+	@echo "now run pytest with the necessary env vars, i.e:"
+	@echo "'$(TEST_ENV) pytest <your-test-file.py>'"
+	@echo "attaching to postgres container, use Ctrl+C detach and stop the container"
+	@echo $$(for i in {1..120}; do printf "-"; done)
+	docker attach postgres-testrunner-postgres
```

Comment:
It used to run tests, and now it's an interactive command. Makefile isn't the best place for interactive shells

Reply:
the reason why I removed the test run is that the whole suite takes over 40 min to run, so in my experience I usually run specific tests rather then the whole thing.